its/projects/IncrementalPRAnalysis/IncrementalPRAnalysis/IncrementalPRAnalysis.csproj

```diff
@@ -1,9 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
What about .NET standard instead of deprecated framework? To make it long-lasting